debian/rules

```diff
@@ -15,7 +15,7 @@ override_dh_auto_test:
 	# nothing to do here, see debian/tests/* instead
 
 override_dh_auto_configure:
-	+pg_buildext configure build-%v --with-extra-version="$${CONF_EXTRA_VERSION:-}" --with-security-flags
+	+pg_buildext configure build-%v --without-pg-version-check --with-extra-version="$${CONF_EXTRA_VERSION:-}" --with-security-flags
```

Comment:
I added this check with the direction of @onderkalaci to be able to build nightlies for pg15